Doc/library/os.rst

```diff
@@ -1386,6 +1399,19 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: posix_openpt(oflag, /)
+
+   Open and return a file descriptor for a master pseudo-terminal device.
+
+   Calls the C standard library function :c:func:`posix_openpt`. The *oflag*
+   argument is used to set file status flags and file access modes as
+   specified in the manual page of :c:func:`posix_openpt` of your system.
```

Comment:
Should the added `O_CLOEXEC` flag be mentioned here?

Reply:
Looks like @gpshead added that to make the returned master fd non-inheritable. According to his commit:

> By default all fds returned by Python APIs must be non-inheritable.

Given this information, I will let you and @gpshead be the judge and I will make the changes accordingly.

Reply:
I'll gladly defer to Greg, but IMO this needs to be the default, and it's not necessary to provide a knob for it (yet?).
But the docs should mention the effect and implementation: the new file descriptor is [non-inheritable]; `O_CLOEXEC` flag is added to *flags*.